v03_pipeline/lib/misc/io.py

```diff
@@ -11,6 +11,44 @@
 B_PER_MB = 1 << 20  # 1024 * 1024
 MB_PER_PARTITION = 128
 
+VETS_SNP_CUTOFF = 0.997
+VETS_INDEL_CUTOFF = 0.99
+VETS_SNP_FILTER = 'high_CALIBRATION_SENSITIVITY_SNP'
+VETS_INDEL_FILTER = 'high_CALIBRATION_SENSITIVITY_INDEL'
+
+
+def annotate_vets(mt: hl.MatrixTable) -> hl.MatrixTable:
+    if not hasattr(mt, 'info') or not hasattr(mt.info, 'CALIBRATION_SENSITIVITY'):
+        return mt
+    return mt.annotate_rows(
+        filters=hl.bind(
+            lambda is_snp, split_cs: (
+                hl.case()
+                .when(
+                    is_snp & (split_cs > VETS_SNP_CUTOFF),
```

Comment:
briefly checked - their header documentation for the vcf just uses '>' and not '>=' so gtg here